docs/route/meta.md

```diff
@@ -250,8 +250,9 @@ If you can't avoid the merge problem with global meta or index routes, we've cre
 [links-export]: ./links
 [use-matches]: ../hooks/use-matches
 [merging-metadata-across-the-route-hierarchy]: #merging-with-parent-meta
-[url-params]: ../guides/routing#dynamic-segments
+[url-params]: ../file-conventions/routes#dynamic-segments
 [root-route]: ../file-conventions/root
-[index-route]: ../guides/routing#index-routes
+<!-- page doesn't exist anymore -->
+[index-route]: ../discussion/02-routes#index-routes
```

Comment:
This section (https://remix.run/docs/en/dev/discussion/02-routes#index-routes) doesn't seem to exist anymore

Reply:
I'd probably just link to https://remix.run/docs/en/main/file-conventions/routes for now.  We can enhance that doc with a sub-heading for index routes in a separate PR if needed.